.env.example

```
@@ -122,3 +122,26 @@ RATE_LIMIT_WINDOW_SECONDS=60
 METRICS_SCRAPE_TOKEN=
 VOICE_WEBHOOK_SECRET=
+
+# ---- PostHog (product analytics) ------------------------------------------
+# Master switch. Both SDKs stay a no-op unless this is true AND a token is set.
+# PostHog never gates auth, billing, compliance or the call path.
+# Server-side (NestJS) kill switch.
+POSTHOG_ENABLED=false
+# Browser (Next.js) kill switch. Separate from POSTHOG_ENABLED on purpose so the
+# browser SDK can be turned off without stopping trusted server-side events.
+# Inlined at build time, so changing it requires a web rebuild.
+NEXT_PUBLIC_POSTHOG_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_ENABLED key should go before the POSTHOG_ENABLED key

(UnorderedKey)